middleware/2fa_service/index.js

```diff
@@ -0,0 +1,4 @@
+const TwoFactorAuthService = require('./TwoFactorAuthService');
+
+// TODO: Provide dependencies / environment variables as constructor arguments?
+module.exports = new TwoFactorAuthService();
```

Comment:
not sure why instead of 2fa.js we now have 3 modules to achieve exactly the same thing (2 of these modules being few lines of code like this one).